pennylane_lightning/core/src/lightning.kokkos.toml

```diff
@@ -0,0 +1,79 @@
+schema = 1
+
+[device]
+name = "lightning.kokkos"
+version = "v0.31.0"
+precision = ['float32', 'float64']
+
+[operations]
+observables = [
+	"NamedObs",
+	"HermitianObs",
+	"TensorObs",
+	"HamiltonianObs",
+]
+
+[[operations.gates]]
+full = [
+	"Identity",
+	"PauliX",
+	"PauliY",
+	"PauliZ",
+	"Hadamard",
+	"S",
+	"T",
+	"PhaseShift",
+	"RX",
+	"RY",
+	"RZ",
+	"Rot",
+	"CNOT",
+	"CY",
+	"CZ",
+	"SWAP",
+	"IsingXX",
+	"IsingXY",
+	"IsingYY",
+	"IsingZZ",
+	"ControlledPhaseShift",
+	"CRX",
+	"CRY",
+	"CRZ",
+	"CRot",
+	"Toffoli",
+	"CSWAP",
+	"MultiRZ",
+]
+# QubitUnitary should never be here.
+# Gates in "matrix" are decomposed to QubitUnitary.
+# Having a QubitUnitary here would mean an infinite loop.
+matrix = [
+	"OrbitalRotation",
+	"SingleExcitation",
+	"SingleExcitationMinus",
+	"SingleExcitationPlus",
+	"MultiControlledX",
```

Comment:
Can you show me where this is? A simple git grep doesn't show any MultiControlledX implementation.

Reply:
We can update the list of supported named operations in these toml files without considering what is and isn't supported in Catalyst. We should revisit this list f support in Catalyst once things are settled.

Reply:
Just to clarify, #516 adds a `controlled_wires` arguments to all quantum gates (just in L-Qubit for now). So in a sense the correct name might again be `__quantum__qis__PauliX` (but with an extra argument) or, say, `__quantum__qis__NCPauliX` (where `NC` is for n-controlled). At the C++ level, this would be like overloading [QuantumDevice](https://github.com/PennyLaneAI/catalyst/blob/main/runtime/include/QuantumDevice.hpp)'s `NamedOperation` and `MatrixOperation` methods with an extra argument `controlled_wires`.

Reply:
https://github.com/PennyLaneAI/catalyst/issues/380